### PR TITLE
What do i need to recover wallet + create table

### PR DESCRIPTION
**Pull request by MarnixCroes**
this question still comes up all the time.
we have similar FAQ listed like _Can I recover my wallet without the password?_
but a general one is missing

I think it would be good to have this with a clear table (for real newbs)
image

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -1072,11 +1072,25 @@ The wallet file backup is sensitive, in terms of privacy, but not critical in te
 So it is good advice to encrypt this wallet file.
 :::
 
+### What do I need to recover my wallet?
+
+To recovery your wallet you need your Password + Recovery words, or Password + Wallet File, as shown in the table below.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:1077]:
```suggestion
To recover your wallet you need either your Recovery Words + Password, or the Wallet File + Password, as shown in the table below.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -1072,11 +1072,25 @@ The wallet file backup is sensitive, in terms of privacy, but not critical in te
...
+
+| | Recovery Words | Wallet File | Password
+:------------ | :-------------:| :-------------:| :-------------:
+Recovery Words |  |  :x: | :heavy_check_mark:
+Wallet File | :x: |   | :heavy_check_mark:
+Password | :heavy_check_mark: |  :heavy_check_mark: | 
+
+:::tip
+If there was an emtpy password (no password) entered at the wallet creation, then the Recovery Words or Wallet File alone are enough to recover the wallet.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:1086]:
```suggestion
If no password was entered at the wallet creation, then the Recovery Words or Wallet File alone are enough to recover the wallet.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -1072,11 +1072,25 @@ The wallet file backup is sensitive, in terms of privacy, but not critical in te
...
 
+### What do I need to recover my wallet?
+
+To recovery your wallet you need your Password + Recovery words, or Password + Wallet File, as shown in the table below.
+
+| | Recovery Words | Wallet File | Password
+:------------ | :-------------:| :-------------:| :-------------:
+Recovery Words |  |  :x: | :heavy_check_mark:
+Wallet File | :x: |   | :heavy_check_mark:
+Password | :heavy_check_mark: |  :heavy_check_mark: |
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:1083]: I would add an X to indicate that the recovery words or the wallet file alone is not enough (ofc assuming that a password was used which should be considered the default)

```suggestion
Recovery Words | :x: | :x: | :heavy_check_mark:
Wallet File | :x: | :x: | :heavy_check_mark:
Password | :heavy_check_mark: | :heavy_check_mark: | :x:
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md (hunk 3 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-UseWasabi.md:1083]: does that make sense?

Comment (yahiheb) [reply docs/FAQ/FAQ-UseWasabi.md:1083]: For me it does.
What does an empty spot mean?

**Pull request merged**